logger.go

```diff
@@ -93,16 +102,26 @@ type severity int
 
 const (
 	sInfo = iota
+	sWarning
 	sError
 	sFatal
 )
 
+const (
```

Comment:
For readability move these above the functions where their used (above initialize)

Reply:
Done.